transaction.sql

```diff
@@ -0,0 +1,60 @@
+-- MySQL dump 10.13  Distrib 8.0.16, for Win64 (x86_64)
+--
```

Comment:
You can remove the sql file as its not needed anymore. We only need the json file